src/rt/aApply.d

```diff
@@ -24,19 +19,18 @@ private import rt.util.utf;
 extern (D) alias int delegate(void *) dg_t;
 
 extern (C) int _aApplycd1(in char[] aa, dg_t dg)
-{   int result;
-    size_t i;
+{
+    int result;
     size_t len = aa.length;
 
     debug(apply) printf("_aApplycd1(), len = %d\n", len);
-    for (i = 0; i < len; )
-    {   dchar d;
-
-        d = aa[i];
+    for (size_t i = 0; i < len; )
+    {
+        dchar d = aa[i];
         if (d & 0x80)
             d = decode(aa, i);
         else
-            i++;
+            ++i;
```

Comment:
You mean optimization wise? It shouldn't nowadays, but it's a good practice, because post-increment __can__ incur extra cost for UDTs.